frontend/app/styles/login.js

```diff
@@ -40,4 +40,4 @@ const style = {
     }
 };
 
-module.exports = style;
+module.exports = styles;
```

Comment:
have you tried ES6 export ?like
 `export default styles`

Since you used import in line 1 :)